src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -333,7 +333,7 @@ function ReportActionItemMessageEdit(props) {
                                 }
                                 openReportActionComposeViewWhenClosingMessageEdit();
                             }}
-                            selection={selection}
+                            selection={!isFocused ? undefined : selection}
```

Comment:
This change is causing where pasting text results in cursor misplacement. This was unable to find as it was blocked by another issue #https://github.com/Expensify/App/issues/22803.

Reply:
Other PR is merged, left comment about the regression caused [here](https://github.com/Expensify/App/issues/21137#issuecomment-1646963074)

Reply:
This caused this regression: https://github.com/Expensify/App/issues/23451